### ISSUE TITLE
 esbenp.prettier-vscode take to long time saving a file

### ISSUE THREAD

**Issue by angelocorreia27** [prettier/prettier-vscode#3112]
Type: <b>Bug</b>

Visual studio code take to long time sanving a file with esbenp.prettier-vscode extension.

Extension version: 10.1.0
VS Code version: Code 1.81.0 (6445d93c81ebe42c4cbd7a60712e0b17d9463e97, 2023-08-02T12:36:11.334Z)
OS version: Linux x64 5.15.0-78-generic
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6200U CPU @ 2.30GHz (4 x 1644)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|2, 3, 3|
|Memory (System)|15.03GB (3.30GB free)|
|Process Argv|. --crash-reporter-id ef6a5f3f-1c15-4377-bd4c-7cb7b70cdda9|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu-wayland|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu-wayland|
|XDG_SESSION_TYPE|wayland|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
vsclangdc:30486549
c4g48928:30535728
dsvsc012:30540252
pynewext54:30695312
azure-dev_surveyone:30548225
vsccc:30803844
2e4cg342:30602488
f6dab269:30613381
showlangstatbar:30737416
vsctsb:30748421
03d35959:30757346
ecj1e332:30736112
pythonfmttext:30731395
pythoncmvfstrcf:30756944
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
gsofa:30804715
pythonnosmt12:30797651
pythonidxpt:30805730
pythonnoceb:30805159
e537b577:30795824
dsvsc013:30795093
dsvsc014:30804076

```

</details>

**Comment by karenpommeroy** [comment]
Bump

Happens on latest VS Code version (1.81.1) as well as couple versions back (the oldest I tried was 1.72).
When saving VS Code popup appears saying "Saving <FILENAME>: Running 'Prettier - Code formatter' Formatter and goes on indefinitely.

Output logs (example saving package.json file from one of my projects):
["INFO" - 10:58:49 PM] Extension Name: esbenp.prettier-vscode.
["INFO" - 10:58:49 PM] Extension Version: 10.1.0.
["INFO" - 10:59:07 PM] Formatting file:///c%3A/Workspace/my/libs/format-gpt/package.json
["INFO" - 11:02:24 PM] Formatting file:///c%3A/Workspace/my/libs/format-gpt/package.json

I noticed that when I open the processed file tab in VS code and then switch to some other open file it does eventually save the file and popup goes away.
